src/modules/routing/messages/KeylistUpdatedMessage.ts

```diff
@@ -4,6 +4,7 @@ import { Expose, Type } from 'class-transformer'
 
 import { AgentMessage } from '../../../agent/AgentMessage'
 import { RoutingMessageType as MessageType } from './RoutingMessageType'
+import { KeylistUpdateMessageOptions } from '..'
```

Comment:
What is this import for?

Reply:
hmm.... good catch, the constructor should take `KeylistUpdatedMessageOptions` not `KeylistUpdateMessageOptions`. I will correct.